lib/msal-core/src/Storage.ts

```diff
@@ -107,18 +107,37 @@ export class Storage {// Singleton
         return results;
     }
 
-    removeAcquireTokenEntries(authorityKey: string, acquireTokenUserKey: string): void {
+    removeAcquireTokenEntries(clearCookies?: boolean): void {
```

Comment:
I would prefer to rename the argument clearCookies to storeAuthStateInCookie to be consistent and clear on the intention.

Reply:
The intention for the argument `clearCookies` is to determine whether or not it should remove related cookies in addition to removing entries from storage.  It'd be strange to have an argument that describes storing data in a function that is not storing data.  Maybe it can be renamed to `removeCookies` or `removeAuthStateCookies`.

It might also be considered that this shouldn't be an optional parameter to begin with, and the function should *always* attempt to remove the related cookies (if they exist) when it's called.

Reply:
@manoj-rath Added a commit to make `removeAcquireTokenEntries` *always* remove cookies, rather than conditionally based on an input argument.